.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This function is for retrying commands in the case they fail. It accepts three arguments
+# $1: Number of retries it will attempt
+# $2: Command to execute
+# $3: Clean up commands
```

Comment:
Should note that 3 is optional, but expected if 2 does something that needs to be cleaned if it fails.